### DISCUSSION TITLE
Erreichbarkeit nach Änderung nach MQTT Erweiterung nobbi1991

### DISCUSSION THREAD

**Discussion by actionbyte** [frickelzeugs/FingerprintDoorbell#107] [Q&A] [answered]
Hi,
ich habe aus dem Abzweig von @nobbi1991 via ElegantOTA im Filesystem die 4 geänderten Dateien auf den ESP geschoben um den MQTT Port ändern zu können.
Funktion vom ESP mit Fingerprint ist noch gegeben, jedoch erreiche ich das WebiF nicht mehr.
Was kann ich jetzt machen?

**Comment by nobbi1991** [comment, marked as answer]
Moin,
Du musst erst das binary bauen. Der ESP kann mit den source files nichts anfangen ;)

Installiere dir VScode mit dem platformIO Plugin. Damit kannst du dann auch direkt flashen.

**Comment by actionbyte** [comment]
Ok danke, hab das mal versucht. Soweit bin ich gekommen, beim Compilieren scheitert es wegen:
UnknownPackageError: Could not find the package with 'ayushsharma82/AsyncElegantOTA @ ^2.2.7' requirements for your system 'windows_amd64'
Bisher habe ich noch nicht herausgefunden wo ich das Paket herbekomme, geschweige denn wie ich es unter platformIO integriere.

**Reply by tilly001** [reply]
You can find it here:

https://github.com/ayushsharma82/AsyncElegantOTA

**Comment by actionbyte** [comment]
Nach einigen Versuchen scheitert es immer wieder an der korrekten Einbindung von dem AsyncElegantOTA. 
Da das Paket ohnehin abgekündigt ist, habe ich mit der GIT Anbindung versucht ElegantOTA zu verwenden, leider bekomme ich hier beim Compilieren einige andere Fehlermeldungen, welche ich nicht beheben kann.
Daher ist 8zumindest für mich dieser Abzweig vom FingerprintDoorbell leider nicht nutzbar.
Inwieweit man (@nobbi1991) den Code für ElegantOTA umschreiben könnte; bleibt für mich mangels Unkenntnis unklar, wäre aber für alle die eben nicht AsyncElegantOTA schon unter VSCode/PlatformIO eringebunden haben; eine große Hilfe. Glaube ich zumindest.

**Comment by andy0172** [comment]
Hab das selbe Problem. Wäre um Hilfe dankbar.
